docs/DEPLOY.md

```diff
@@ -0,0 +1,53 @@
+
+## Deploy
+Deployment of team web on team10-20.studenti.fiit.stuba.sk.
+
+#### Steps
+1. Build the app locally:
+
+	```shell script
+	npm run build
+	```
+	This creates /dist folder which content is uploaded on server. 
+
+1. Connect to server using ssh:
+
+	```shell script
+	ssh -i team10.key ubuntu@team10-20.studenti.fiit.stuba.sk
+	```
+
+1. Setup sftp server in FileZilla via File -> Site Manager -> New site:
+
+    Protocol: sftp
+
+    Host: ubuntu@team10-20.studenti.fiit.stuba.sk
+
+	Logon Type: Key file
+    
+	User: ubuntu
+    
+	Key file: the key needs to be converted into a supported format (team10key.ppk)
```

Comment:
Is this connected to FileZilla or is it common for all `SFTP client`s? If so, could you please provide more info how to convert the private key into the supported format?